dotnet/src/dotnetframework/GxClasses/Model/GXWebProcedure.cs

```diff
@@ -138,6 +141,7 @@ protected bool initPrinter(String output, int gxXPage, int gxYPage, String iniFi
 			string idiom;
 			if (!Config.GetValueOf("LANGUAGE", out idiom))
 				idiom = "eng";
+			fileContentInline = true;
```

Comment:
This code is about PDF Procedures. Maybe it mixed up from another branch?

Reply:
You are right. Moved this to https://github.com/genexuslabs/DotNetClasses/pull/826.